shared/src/db.json

```diff
@@ -5,7 +5,7 @@
       "name": "Tomato pasta",
       "quantity": 1,
       "availableFromDate": "1605802429",
-      "expiryDate": "1405802429",
+      "expiryDate": "1607849537537",
       "info": "Vegetarian",
```

Comment:
These timestamps were in the 70s so was the whole list was filtered out.  Have updated 2 of them so that the filter system will still have something to run on.

Reply:
Depends 🤔 if you are converting using seconds or milliseconds.
For sure it was in the past but that expiry date is in 2014.

We will need to make sure we align how we are creating these timestamps from the addMeal screen so we don't end up with botched dates on one platform or another.  Let's catch up tomorrow morning